### DISCUSSION TITLE
[4.1]Attempt to test embedded media and cron

### DISCUSSION THREAD

**Discussion by HDInfautre** [joomla/joomla-cms#35873] [Q&A] [answered]
hello

This is the 1st time I try to test an alpha version without being a developer.
I have read https://developer.joomla.org/news/862-joomla-4-1-alpha-1-discover-what-s-inside.html and a number of PRs.
Where I don't get it is that I downloaded the alpha1 version focusing on 2 topics of interest and I don't see how to make them work, even with the help of the PR sometimes. It seems to me that these 2 topics should be integrated in the 4.1 alpha version?

Thank you for your indulgence for this intrusion by helping me test what I mention or telling me how to make a suggestion for improvement in an existing PR (not the skill to create one) if it does not exist?

**1/ integration of embded media**
The most important is the simple way by copy / paste to insert a video file (youtube / dailymotion ...) or audio (soundcloud ...) to make the addition in an article joomla ultra-simple (what do many other cms. on the other hand it is a real problem for novices that I know). 
I arrived that with plugins under joomla 4.0.3 but I understood that it became native in joomla 4.1!
On the other hand, in my attempt to integrate a pdf file. I see that it has the same dimensions in the blog and the article while it would be necessary that the dimensions are different in the blog / article part; Is there such a thing but I have not seen it? is it a suggestion to make?

**2/ cron**
Again I tried to follow :
https://github.com/joomla/joomla-cms/pull/25022
https://github.com/joomla/joomla-cms/pull/35143
I don't see the plugin to activate to make this test.

Regards

**Comment by HDInfautre** [comment]
Hello
I installed the alpha2 version and I don't see any change.
Thank you for helping me test joomla 4.1!?

**Comment by HDInfautre** [comment]
+1

**Comment by bembelimen** [comment, marked as answer]
Hi @HDInfautre if you want to test features which are not merged yet (read: the PR is still in [the list of open PRs](https://github.com/joomla/joomla-cms/pulls)), you have to download and install it manually. There are two ways:
1. with the Patch Tester ([more information](https://docs.joomla.org/Component_Patchtester_for_Testers))
2. by scrolling down to the testing area in the PR and use the download button (there are then links to download the complete Joomla! + PR):
![grafik](https://user-images.githubusercontent.com/1229869/140962230-1d8ef412-367b-4b36-95ae-55ec86d9d15c.png)
